pkg/cache/cache.go

```diff
@@ -27,55 +32,55 @@ var (
 
 type localCache struct{}
 
-func (localCache) getDecision(clientIP string) (bool, error) {
-	banned, isCached := cache.Get(clientIP)
-	bannedString, isValid := banned.(string)
-	if isCached && isValid && len(bannedString) > 0 {
-		return bannedString == cacheBannedValue, nil
+func (localCache) get(key string) (string, error) {
+	value, isCached := cache.Get(key)
+	valueString, isValid := value.(string)
+	if isCached && isValid && len(valueString) > 0 {
+		return valueString, nil
 	}
-	return false, fmt.Errorf(CacheMiss)
+	return "", fmt.Errorf(CacheMiss)
```

Comment:
```suggestion
	return _, fmt.Errorf(CacheMiss)
```
Dunno if it's used later by as an error we don't really need to return the "value"

Reply:
it's not possible actually to return _
maybe a default constant value "empty"

Reply:
"" is already empty